docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -1,9 +1,10 @@
-include::./attributes.adoc[]
 = {project-name} - Writing Your Own Extension
-
 :numbered:
 :sectnums:
 :sectnumlevels: 4
+:toc:
```

Comment:
We will need to check how it looks like on the website.

Reply:
We should IMHO add TOC to all guides - recently, I created https://github.com/quarkusio/quarkus/issues/1490